pyne/material.pyx

```diff
@@ -608,7 +608,9 @@ cdef class _Material:
 
 
     def decay_heat(self):
-        """This provides the decay heat using the comp of the the Material.
+        """This provides the decay heat using the comp of the the Material. It
+        assumes that the composition of material is given in units of [grams]
+        and returns decay heat in units of [MW/material object total mass].  
```

Comment:
I think the decay heat is returned in  units of MW: https://github.com/pyne/pyne/blob/develop/src/material.h#L237

Reply:
I think line 237 is referring to the activity function, https://github.com/pyne/pyne/blob/develop/src/material.h#L239 says that the decay heat function returns dose per gram, which is synonymous with [MW/material object total mass in grams]. I will correct my PR to include grams.

Reply:
Hi @gwenchee - doxygen/javadoc style docstrings are placed above the function they are documenting, not below.  So 237 is the correct line.  Also, if you see 239 has units of mrem/h, which is more appropriate for dose, than decay heat.

Reply:
Yup that makes sense. I changed it to reflect that the unit is [MW]